docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
       - testbed
     volumes:
       - .:/usr/src/app
-      - /usr/src/app/node_modules
+      - ./node_modules:/usr/src/app/node_modules
```

Comment:
How did you realize you needed to add this volume mount? Adding this breaks for me since I have no node_modules on my host, so I think you're running `npm install` on your side and then loading the node_modules into the container. 

Is there any reason for doing this over having node_modules live only inside the container?

Reply:
volume - yes, I'm now running it by using the npm run dev

Reply:
if you don't do the mount, you will have to rebuild the image everytime. Otherwise the change in new package will not be added to the image. So in order to make it working in dev mode and be able to make changes in modules, one needs to mount the node_modules

Reply:
That makes sense. I didn't see myself modifying packages often enough to care that I needed to rebuild each time and to be totally honest, this was just one of the tips I found on the internet for using docker in a react project so I wasnt sure what's more realistic.

We'll keep this change then. Do the `npm install` on the host and mount the `node_modules` in the container